garage/tf/algos/npo.py

```diff
@@ -17,18 +17,19 @@
 from garage.tf.misc.tensor_utils import flatten_batch_dict
 from garage.tf.misc.tensor_utils import flatten_inputs
 from garage.tf.misc.tensor_utils import graph_inputs
-from garage.tf.optimizers import LbfgsOptimizer
+from garage.tf.optimizers import PenaltyLbfgsOptimizer
 
 
 @unique
 class PGLoss(Enum):
     VANILLA = "vanilla"
-    CLIP = "clip"
+    SURROGATE_VANILLA = "surrogate_vanilla"
```

Comment:
I think just `SURROGATE`/`"surrogate"` is fine here

Reply:
The shorter the better